ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_norms.py

```diff
@@ -1,10 +1,40 @@
 # global
-from hypothesis import strategies as st
+from hypothesis import given, strategies as st
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.helpers import handle_test
 
+@handle_test(
+    fn_tree="functional.ivy.l1_normalize",
```

Comment:
Sorry I missed this, this should be called from experimental Ivy

Reply:
Thanks for pointing this out ! Fixed it :smile: ! Kindly check